hil/cli/node.py

```diff
@@ -11,22 +13,70 @@ def node():
 
 @node.command(name='list')
 @click.argument('pool', type=click.Choice(['free', 'all']), required=True)
-def nodes_list(pool):
+@click.option('--jsonout', is_flag=True)
+def nodes_list(pool, jsonout):
```

Comment:
I feel like it would look better if in the table we say `ALL NODES` or `FREE NODES` since we used to do that in the CLI.  Regardless of free or all it says NODE LIST right now.  (I'm not super adamant about it though, curious what other folks think)